internal/demo/helloworld/helm/helloworld/values.yaml

```
@@ -9,6 +9,9 @@ temporal:
   # Connection details (required if connectionName is empty)
```

```suggestion
  # Connection details (either mtlsSecretName or apiKey is required if connectionName is empty)
```
(Or feel free to say that only one or the other is required in another way)